src/Microsoft.CSharp/ref/Microsoft.CSharp.csproj

```diff
@@ -2,6 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
+    <!-- must match version supported by frameworks which support 4.0.* inbox -->
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
```

Comment:
If we do add more APIs then should we jump to whatever version is in the dir.props and harvest the 4.0.0.0 version?